support/cas-server-support-ehcache-ticket-registry/src/main/java/org/apereo/cas/config/EhcacheTicketRegistryConfiguration.java

```diff
@@ -41,6 +42,7 @@
  */
 @Configuration("ehcacheTicketRegistryConfiguration")
 @EnableConfigurationProperties(CasConfigurationProperties.class)
+@ConditionalOnProperty(prefix = "cas.ticket.registry.ehcache", name = "enabled", havingValue = "true", matchIfMissing = true)
 @Slf4j
 public class EhcacheTicketRegistryConfiguration {
```

Comment:
I think it might be good to deprecate this class, and add a startup warning similar to what's done with google-apps.